gulpfile.js

```diff
@@ -571,6 +575,7 @@ export const makeZip = gulp.series(release, async function () {
       "Build/Cesium/**",
       "Build/CesiumUnminified/**",
       "Build/Documentation/**",
+      "Build/Specs/**",
```

Comment:
We also need to include the `Specs` for the packages. That aside, the motivation for including the Spec files (instead of the bundles `SpecList.js` that allows running of the tests from the ZIP file) is a little unclear to me.

Reply:
This is needed to run the test commands from within the packaged release zip, which was the previous behavior and was discussed in https://github.com/CesiumGS/cesium/issues/3911. I

I was able to execute the top-level tests, both in browser after running `npm start` and on the command line after running `npm test` with this inclusion. 

This just includes the built specs ready for execution, right?

Reply:
Yes, you're right, sorry for the confusion. What I meant to ask is whether or not the root level `Specs` folder should be included given that we're not including the `Specs` from the workspaces.

Reply:
Since the complete spec list is used as the entry point to create the bundle, the top-level specs have all the code needed to run all the specs together, correct?